client/data/authorizations/resolvers.ts

```diff
@@ -59,19 +59,19 @@ export function* getAuthorization(
 			const result = yield apiFetch( {
 				path: `${ NAMESPACE }/authorizations/${ paymentIntentId }`,
 			} );
-			const isCaptured = result
-				? ( result as GetAuthorizationApiResponse ).is_captured
-				: false;
+
+			const {
+				is_captured: isCaptured,
+			} = result as GetAuthorizationApiResponse;
 
 			yield updateAuthorization( {
-				payment_intent_id: ( result as GetAuthorizationApiResponse )
-					.payment_intent_id,
+				payment_intent_id: paymentIntentId,
```

Comment:
I think we missed destructuring `paymentIntentId` from `result` above.

Reply:
Yep, did small refactor here [8fe9dc5](https://github.com/Automattic/woocommerce-payments/pull/5006/commits/8fe9dc5b648af7da66ce7d7355a38ca22d53595d)